resources/public/index.html

```diff
@@ -28,24 +19,12 @@
     </head>
     <body style="margin:0">
         <div id="app">
-	  <div style="display:flex;justify-content:space-around">
-	    <img src="/image/spiral.gif" style="height:200px;width:200px;margin-top:200px">
-	  </div>
-	</div>
-
+			<div style="display:flex;justify-content:space-around">
+				<img src="/image/spiral.gif" style="height:200px;width:200px;margin-top:200px">
+			</div>
+		</div>
         <script src="/js/compiled/orcpub.js"></script>
-	<link rel="stylesheet" href="/font-awesome-4.7.0/css/font-awesome.min.css">
-	<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
-
-	<script>
-	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-	  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-	  ga('create', 'UA-69209720-3', 'auto');
-	  ga('send', 'pageview');
```

Comment:
Same as previous comment about wrapping in config vs removal.

Reply:
Same as above, GA when uberjar is built, it in a different place (and yes THAT needs to be configurable )

Reply:
The ideal fix would be move [the css here](https://github.com/johnliss/orcpub/blob/develop/src/clj/orcpub/index.clj#L31) to a garden config.

I just haven't figured out how to do that.